libs/core/src/lib/tree/components/tree-item/tree-item.component.html

```diff
@@ -0,0 +1,86 @@
+<div
+    class="fd-tree__item-container"
+    tabindex="0"
+    (click)="_onClick($event)"
+    (keyup.enter)="_onClick($any($event))"
+    (keyup.space)="_onClick($any($event))"
+    fdkSelectableItem
+    (selected)="_setSelected($event)"
+    [value]="value"
+    #itemContainer
+    [ngClass]="[
+        state ? 'has-highlight-indicator has-highlight-indicator--' + state : '',
+        navigatable ? 'fd-tree__item-container--active' : '',
+        wrapContent ? 'fd-tree__content--wrap' : ''
+    ]"
+>
+    <ng-container *ngIf="_selectionModel?.placement === 'left'">
+        <ng-template [ngTemplateOutlet]="selection"></ng-template>
+    </ng-container>
+    <ng-container *ngIf="hasChildren">
+        <button
+            tabindex="-1"
+            *ngIf="children.length > 0 || hasProjectedChildren"
+            (click)="_toggleExpandState($event)"
+            class="fd-tree__expander"
+            [class.is-expanded]="expanded"
+            aria-haspopup="true"
+            [attr.aria-expanded]="expanded"
+        >
+            <fd-icon [glyph]="expanded ? 'navigation-down-arrow' : 'navigation-right-arrow'"></fd-icon>
+        </button>
+        <div class="fd-tree__expander fd-tree__expander--loading" *ngIf="!_childrenLoaded">
+            <fd-skeleton type="rectangle" width="1rem" height="1rem"></fd-skeleton>
+        </div>
```

Comment:
why only the expander has a skeleton? Shouldn't the whole item have it?

Reply:
@InnaAtanasova Item skeleton is located here: https://github.com/SAP/fundamental-ngx/pull/9781/files#diff-6527bb81695f8c236de4e9413a47f48fd6eddda06a95fbe0927743b91f5c71e9R7 since it rendered before we initiate tree item